src/components/presentational/MealLog/MealLog.stories.tsx

```diff
@@ -44,8 +51,14 @@ export const Default = {
         previewState: {
             name: 'state',
             control: 'radio',
-            options: ['loading', 'no data', 'with data']
-        }
-    },
-    render: render
+            options: ['loading', 'no data', 'with data', 'live']
+        },
+        showMealNumbers: {
+            name: 'show meal numbers?'
+        },
+        highlightSelectedMeal: {
+            name: 'highlight selected meal?'
+        },
```

Comment:
I guess I look at the Storybook as more of a dev + non-dev showroom of components.  I think design folks and other non-devs should be able to navigate the Storybook and explore things in a fairly human readable way.  Having the actual camel cased property names just feels a little terse and unpolished to me.  I think it might to most non-dev as well.

I also find that, particularly with more complex components, the Storybook controls don't always align 1:1 with the component properties anyways.  Sometimes you may want to configure a component with a particular property or set of properties based on the value of one or a combination of control flags or states.

I cringe a little when I see the storybook controls that have arrays of JSON in them.  I don't think that many users, particularly non-devs, will ever want to fuzz with those values to adjust the display.  Even devs can have a hard time with that since the data type definitions aren't available there.  I think having a selector with things like "scenario A" and "scenario B" that translate to some combination of properties and data being configured on the component make the Storybook easier to use for non-devs.

My approach has generally been to try to make the Storybook controls as user friendly and non-technical as possible, since devs have the ability to look into the code to see how the Storybook controls translate to real properties when they need to use them.  

All that said, if the consensus is that we should be using it more as a dev reference with the exact property names, I'm not averse to that.  I think it may just add some friction for other types of Storybook users.